numpy/random/tests/test_extending.py

```diff
@@ -88,8 +89,8 @@ def test_cython(tmp_path):
     assert values.shape == (10,)
     assert values.dtype == np.float64
 
-@pytest.mark.skipif(numba is None or cffi is None,
-                    reason="requires numba and cffi")
+@pytest.mark.skipif(numba is None or cffi is None or numba.__version__,
```

Comment:
`numbq.__version__` all by itself?

Reply:
:facepalm: sorry, first thought I would add a version check, then opted for just catching the SystemError as a simpler way.